nautobot_ssot_aristacv/signals.py

```diff
@@ -93,11 +95,14 @@ def post_migrate_create_manufacturer(apps=global_apps, **kwargs):
 
 def post_migrate_create_platform(apps=global_apps, **kwargs):
     """Callback function for post_migrate() -- create Arista Platform."""
+    plugin_settings = settings.PLUGINS_CONFIG["nautobot_ssot_aristacv"]
     Platform = apps.get_model("dcim", "Platform")
     Manufacturer = apps.get_model("dcim", "Manufacturer")
     Platform.objects.get_or_create(
-        name="arista.eos.eos",
-        slug="arista_eos",
-        napalm_driver="eos",
-        manufacturer=Manufacturer.objects.get(slug="arista"),
+        slug=slugify(plugin_settings["platform_name"]),
```

Comment:
Why are we changing the slug here? I believe it should always be the netmiko OS name.

Reply:
The goal was to set the slug programmatically rather than hardcoding it in order to facilitate the use case I described in your other comment.

Reply:
Perhaps we should just allow setting the slug and the platform name then?

Reply:
What would be a use case to have a slug that is not auto-derived from the name?

Reply:
When you want to use a more colloquial name but still need the slug to be the netmiko driver. I can see situations where a user would want to have the name simply be EOS but we need the slug to still be arista_eos.

Reply:
I see where you're coming from now. What I'm not sure of is whether we should use the slug field to store information about the netmiko driver name. I have two ideas on how to address this:
- Utilize netutils to translate the present Platform field for the NAPALM name into the netmiko driver name
- OR - since this is by the very nature of it an Arista-only plugin - would it be wrong to hard-code the netmiko driver name in the code somewhere, possibly as a constant?

Reply:
I'm specifically thinking of the Golden Config App where it uses the slugs to map to the parsers. Personally, I think it makes sense to have the slug always be the netmiko driver and NAPALM driver is obvious. I've resorted to using the Platform name for the Ansible collection, thus the `arista.eos.eos` prior, but that could be addressed with a new attribute on the Platform model in the future.

Reply:
my thought would be to do not overload the meaning of the `slug` and leave this to the `napalm_driver` attribute. If the netmiko driver is different then, maybe we should consider adding some new attribute or using some mapping derived from the `napalm_driver`.
My concern, and as you say, not specific to this project, is that using an attribute (`slug`) for other purposes (mapping to netmiko drivers) will end up with more confusion.
Is only Golden Config App which does this usage? or there are more applications?

Reply:
@jdrew82 did you get the chance to explore this more yet? Should we have a quick synchronous discussion?

Reply:
@Kircheneer there was an internal discussion and it appears that a large number of apps are using the slug to determine action and specifically need it to be the netmiko driver. I 100% agree that we should add that as an attribute on the Platform object but until that's done, we need to leave the slug as the netmiko driver.